src/surface_conditions/surface_conditions.jl

```diff
@@ -27,14 +27,9 @@ function update_surface_conditions!(Y, p, t)
     wrapped_sfc_setup = sfc_setup_wrapper(sfc_setup)
     sfc_temp_var =
         p.atmos.surface_model isa PrognosticSurfaceTemperature ?
-        (; sfc_prognostic_temp = Fields.field_values(Y.sfc.T)) : (;)
+        Fields.field_values(Y.sfc.T) : nothing
```

Comment:
Would this make it harder if we want to add more prognostic variables to `Y.sfc` in the future?

Reply:
I don't think so, we could also just pass `Fields.field_values(Y.sfc)`, which will give us all of them. I mostly wanted to just get rid of the splatting